index.html

```diff
@@ -2828,6 +2948,12 @@ <h3>Specifying sort and search functionality</h3>
 	<p><a href="https://www.w3.org/TR/charmod/#sec-CollationUnits">Units of collation, C008</a>, in <cite>Character Model for the World Wide Web: Fundamentals</cite></p>
 	</details>
 	</div>
+-->
+
+	<aside class="warning" id="find-is-hard" title="String search is hard">
```

Comment:
As a warning about _string searching_ this seems a little oddly placed here.  I think that similar difficulties apply to matching strings for sorting, but i think it would be useful to couch this warning in those terms.

Reply:
This section is (and has historically been) called "specifying sort and search functionality". All of the prose in the section is about _sorting_. This is the only mention of "search"--I added it so that we wouldn't have a gap. I think maybe a subsection div above it would help. I also retitled as "under construction". See the new PR